.prettierignore

```diff
@@ -7,15 +7,23 @@
 !*/
 
 # Allow certain file types
+!*.cjs
+!*.js
 !*.json
 !*.md
 !*.mdx
+!*.mjs
+!*.ts
+!*.tsx
 !*.yml
 # TODO: add more types
 
 # Keep license files as is
 LICENSE.md
 
+# Skip autogenerated files tracked by git
+/packages/engine/src/worker/runner/javascript/hash_stdlib.js
```

Comment:
@Alfred-Mountfield this file was obviously generated, so I excluded it from the Prettier list.

<img width="1242" alt="Screenshot 2022-01-12 at 13 07 39" src="https://user-images.githubusercontent.com/608862/149146359-63955976-85b4-4650-ab1f-b6c6198f66ec.png">

I wondered if we could remove the file from git, but I could not find how it got generated. Never mind for now I guess.

Are there any other JS files we don’t want to cover by Prettier?

Reply:
Copying in @litvand

Reply:
`bundle_arrow.js` can also be skipped. Nothing else comes to mind.

Reply:
Good point, probably worth reverting changes to the file to avoid such a big diff

Reply:
✅ 85221008d3dd385e53351ed57e187be5dab13aac